docs/dev-tools/zombienet/tutorials/minimal-example.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Minimal Example of the usage of Zombienet
+description: This tutorial provides an example of how to use Zombienet to spawn a basic network and run a simple test over it.
+---
+
+# A Minimal Example of the usage of Zombienet
+
+This tutorial provides an example of how to use Zombienet to spawn a basic network and run a simple test against it.
+
+## Prerequisites
+
+To follow this tutorial, first, you need to have Zombienet installed. If you haven't done so, please follow the instructions in the [Installation](../overview.md/#installation){target=_blank} section.
+
+## Defining the network
+
+As mentioned in the [Configuration Files](../overview.md/#configuration-files){target=_blank} section, Zombienet uses a configuration file to define the ephemeral network that will be spawned. To follow this tutorial, create a file named `minimal-example.toml` with the following content:
+
+```toml
+[settings]
+timeout = 120
+
+[relaychain]
+
+[[relaychain.nodes]]
+name = "alice"
+validator = true
+
+[[relaychain.nodes]]
+name = "bob"
+validator = true
+
+[[parachains]]
+id = 100
+
+  [parachains.collator]
+  name = "collator01
+```
+
+This configuration file defines a network with a relaychain with two nodes, `alice` and `bob`, and a parachain with a collator named `collator01`. Also, it sets a timeout of 120 seconds for the network to be ready.
+
+## Running the network
+
+To spawn the network, run the following command:
+
+```bash
+zombienet -p native spawn minimal-example.toml
+```
+
+This command will spawn the network defined in the `minimal-example.toml` configuration file. The `-p native` flag specifies that the network will be spawned using the native runtime.
+
+If successful, you will see the following output:
+
+<div id="termynal" data-termynal>
```

Comment:
Marking this item as solved since it will be resolved when those folders are merged into the main branch. It will be addressed at https://opslayer.atlassian.net/browse/DRP-40